projects/packages/videopress/src/client/block-editor/blocks/video/components/poster-panel/index.native.js

```diff
@@ -0,0 +1,11 @@
+export const isVideoFramePosterEnabled = () =>
+	!! window?.Jetpack_Editor_Initial_State?.available_blocks?.[ 'v6-video-frame-poster' ];
```

Comment:
I kept this function as it's used in the `saved` component:
https://github.com/Automattic/jetpack/blob/9608824e67a60f3550b57fe43c436e046f1a792f/projects/packages/videopress/src/client/block-editor/blocks/video/save.tsx#L13